### PR TITLE
build(package.json): Upgrade to React 16.8

### PR DESCRIPTION
**Pull request by Laviro**
**What**: Upgrade to React 16.8 to use its new features.

### CODE REVIEW

File: packages/patternfly-3/patternfly-react/package.json [hunk 1]
```diff
@@ -48,8 +48,8 @@
   },
   "peerDependencies": {
     "prop-types": "^15.6.0",
-    "react": "^16.3.1",
-    "react-dom": "^15.6.2 || ^16.2.0"
+    "react": "^16.8.1",
```
Comment (ohadlevy) [inline packages/patternfly-3/patternfly-react/package.json]: as its peer Dependencies, I think this implies every patternfly user need to upgrade react too....not sure if we are ready for it?

Comment (Laviro) [reply packages/patternfly-3/patternfly-react/package.json]: thanks, removed all peer Dependencies changes

**Comment by redallen** [comment]
It may be wise to upgrade to React 16.04 first (and use this same version across all packages). There are a few unit tests I'm currently unable to debug in #1408 .

**Comment by Laviro** [comment]
Sorry, have not much time to work on this right now,
does someone want to work on an upgrade to 16.4 as @redallen suggested?

**Pull request closed without merging**